zine/server/watcher/LinuxWatcher.zig

```diff
@@ -100,12 +317,35 @@ pub fn listen(
 
                     log.debug("ISDIR CREATE {s}", .{dir_path});
 
-                    try self.addTree(gpa, parent.kind, dir_path);
+                    const new_fd = try self.addTree(gpa, parent.kind, dir_path);
+                    try self.addChild(gpa, event.wd, new_fd);
+                    const data = self.watch_fds.get(new_fd).?;
+                    switch (data.kind) {
+                        .input => {
+                            reloader.onInputChange(data.dir_path, "");
+                        },
+                        .output => {
+                            reloader.onOutputChange(data.dir_path, "");
+                        },
+                    }
```

Comment:
This is not directly related to move, but `input` moves are met with `output` creates.